### ISSUE TITLE
How to Generate and Use `cam_data_argoverse.pt`

### ISSUE THREAD

**Issue by dachii-azm** [alexanderswerdlow/BEVGen#11]
Thank you for your outstanding work and for sharing this repository. I am currently trying to run inference with the Argoverse dataset. During the setup, I learned that I need to generate `cam_data_argoverse.pt`.

I was able to generate this file using the `save_cam_data` function in `multi_view_generation/bev_utils/argoverse.py`. However, I have a few questions:

1. Do I need to generate `cam_data_argoverse.pt` for all the data in the dataset?
2. Since `save_cam_data` generates the file in batches, running it multiple times causes the` cam_data_argoverse.pt` file to be overwritten. Should I modify the code to append data for each batch, or is there a recommended way to handle this?
Additionally, I noticed that the `cam_data_nuscenes.pt` file shared in the training branch was generated with a batch size of 2. Is this file shared as an example or demo of the process, or was it generated for training all data?

Any guidance would be greatly appreciated.

To run all data, I wrote this code.
```
cam_data = {}
for i, batch in enumerate(tqdm(batch_iter)):
    for k, v in batch.items():
        if "intrinsics" in k or "extrinsics" in k:
            if k in cam_data:
                cam_data[k] = torch.cat((v, cam_data[k]), dim=0)
            else:
                cam_data[k] = v
torch.save(cam_data, 'cam_data_argoverse.pt')
```

**Comment by alexanderswerdlow** [comment]
Hi! 

I believe `cam_data_argoverse.pt` is fixed over the dataset. I only trained on 3 cameras from argoverse and these should be in the same locations so I just saved it to file once. Thus, just one sample should be sufficient.

It's used in `get_image_direction_vectors` where there isn't real batch dim and will apply to all elements passed through the model.

Hope this helps!

**Comment by dachii-azm** [comment]
Thank you for your clear explanation. 
I now understand the process, and I was able to successfully run the inference code. 
I really appreciate your support!
